cmake/Modules/Findlibrealsense.cmake

```diff
@@ -0,0 +1,52 @@
+###############################################################################
+# Find librealsense
+#
+#     find_package(librealsense)
+#
+# Variables defined by this module:
+#
+#  LIBREALSENSE_FOUND                 True if librealsense was found
+#  LIBREALSENSE_INCLUDE_DIRS          The location(s) of librealsense headers
+#  LIBREALSENSE_LIBRARIES             Libraries needed to use librealsense
+
+if (LIBREALSENSE_LIBRARIES AND LIBREALSENSE_INCLUDE_DIRS)
+  # in cache already
+  set(LIBREALSENSE_FOUND TRUE)
+else (LIBREALSENSE_LIBRARIES AND LIBREALSENSE_INCLUDE_DIRS)
+  find_path(LIBREALSENSE_DIR include/librealsense
+            PATHS $ENV{LIBREALSENSE_DIR} NO_DEFAULT_PATH)
+  if (WIN32)
+    set(LIBREALSENSE_RELEASE_NAME realsense.lib)
+    set(LIBREALSENSE_DEBUG_NAME realsense-d.lib)
+    set(LIBREALSENSE_SUFFIX Win32)
+    if (CMAKE_SIZEOF_VOID_P EQUAL 8)
+      set(LIBREALSENSE_SUFFIX x64)
+    endif ()
+  else (WIN32)
+    set(LIBREALSENSE_RELEASE_NAME librealsense.so)
+    set(LIBREALSENSE_DEBUG_NAME librealsense.a)
+  endif (WIN32)
+
+find_path(LIBREALSENSE_INCLUDE_DIR
+          NAMES librealsense
+          PATHS /usr/local/include /usr/include /opt/local/include /sw/include ${LIBREALSENSE_DIR}/include NO_DEFAULT_PATH)
+
+find_library(LIBREALSENSE_LIBRARY
+             NAMES ${LIBREALSENSE_RELEASE_NAME}
+             PATHS ${LIBREALSENSE_DIR}/bin/ /usr/local/lib /usr/lib /opt/local/lib /sw/lib ${LIBREALSENSE_DIR}/lib/ NO_DEFAULT_PATH
+             PATH_SUFFIXES ${LIBREALSENSE_SUFFIX})
+find_library(LIBREALSENSE_LIBRARY_DEBUG
+             NAMES ${LIBREALSENSE_DEBUG_NAME}
+             PATHS ${LIBREALSENSE_DIR}/bin/ ${LIBREALSENSE_DIR}/bin/debug/ NO_DEFAULT_PATH
```

Comment:
Thank you for support static link.

Why different search path in Release and Debug? (Findlibrealsense.cmake #L36 #L40)
I think that should remove <code>${LIBREALSENSE_DIR}/bin/debug/</code> from search path of Debug library. And should add <code>${LIBREALSENSE_DIR}/lib/</code>.

What do you think?